### PR TITLE
[Student] login QR fix for tests

### PR DESCRIPTION
**Pull request by Blackchai**

### CODE REVIEW

File: libs/login-api-2/src/main/java/com/instructure/loginapi/login/activities/BaseLoginLandingPageActivity.kt [hunk 1]
```diff
@@ -39,6 +39,7 @@ import com.google.gson.Gson
 import com.instructure.canvasapi2.apis.ErrorReportAPI
 import com.instructure.canvasapi2.models.ErrorReportPreFill
 import com.instructure.canvasapi2.utils.*
+import com.instructure.loginapi.login.BuildConfig
```
Comment (joehoag) [inline libs/login-api-2/src/main/java/com/instructure/loginapi/login/activities/BaseLoginLandingPageActivity.kt]: Unnecessary for now?

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/LoginLandingPage.kt [hunk 2]
```diff
@@ -17,6 +17,8 @@
 package com.instructure.student.ui.pages
```
Comment (joehoag) [inline apps/student/src/androidTest/java/com/instructure/student/ui/pages/LoginLandingPage.kt]: The changes in this file seem unnecessary for now.

Comment (Blackchai) [reply apps/student/src/androidTest/java/com/instructure/student/ui/pages/LoginLandingPage.kt]: yup yup! Just missed that assert = true

File: libs/login-api-2/src/main/java/com/instructure/loginapi/login/activities/BaseLoginLandingPageActivity.kt (hunk 1 above)
Comment (Blackchai) [reply libs/login-api-2/src/main/java/com/instructure/loginapi/login/activities/BaseLoginLandingPageActivity.kt]: Roger!

**Pull request merged**